src/CMakeLists.txt

```diff
@@ -641,7 +641,8 @@ set(ceph_common_deps
   ${BLKIN_LIBRARIES}
   ${CRYPTO_LIBS}
   ${CMAKE_THREAD_LIBS_INIT}
-  ${CMAKE_DL_LIBS})
+  ${CMAKE_DL_LIBS}
+  ${UDEV_LIBRARIES})
```

Comment:
@liewegas 
Not available on FreeBSD

Reply:
right.. let see if I can sort out how to guard these

Reply:
pushed a patch fixing it.. look ok?